src/control/cmd/daos/filesystem.go

```diff
@@ -265,3 +269,151 @@ func (cmd *fsGetAttrCmd) Execute(_ []string) error {
 
 	return nil
 }
+
+type fsCheckCmd struct {
+	existingContainerCmd
+
+	FsckFlags FsCheckFlag `long:"flags" short:"f" description:"comma-separated flags: print, remove, relink, or verify"`
+	DirName   string      `long:"dir-name" short:"n" description:"directory name under lost+found to store leaked oids (a timestamp dir would be created if this is not specified)"`
+}
+
+func (cmd *fsCheckCmd) Execute(_ []string) error {
+	ap, deallocCmdArgs, err := allocCmdArgs(cmd.Logger)
+	if err != nil {
+		return err
+	}
+	defer deallocCmdArgs()
+
+	if err := cmd.resolveContainer(ap); err != nil {
+		return err
+	}
+
+	var cleanupPool func()
+	cleanupPool, err = cmd.connectPool(C.DAOS_PC_RW, ap)
```

Comment:
```suggestion
	cleanupPool, err := cmd.connectPool(C.DAOS_PC_RW, ap)
```

Reply:
cmd/daos/filesystem.go:292:19: no new variables on left side of :=

Reply:
ah nevermind. missed removing the line before